### PR TITLE
Added app management functionality.

### PR DESCRIPTION
**Pull request by rtorres87**

### CODE REVIEW

File: src/main/java/com/plenigo/sdk/models/CustomerAppRequest.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
+package com.plenigo.sdk.models;
+
+
+/**
+ * <p>
+ * This class is used for requesting an app token for customer app management purposes.
```
Comment (maximilianschweitzer) [inline src/main/java/com/plenigo/sdk/models/CustomerAppRequest.java]: Really? ;-)

File: src/test/java/com/plenigo/sdk/services/AppManagementServiceTest.java [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+/**
+ * Tests for {@link com.plenigo.sdk.services.ProductService}.
+ */
+//@RunWith(PowerMockRunner.class)
+//@PrepareForTest({EncryptionUtils.class, PlenigoManager.class, RestClient.class})
+public class AppManagementServiceTest {
+
+    @Before
+    public void setup() throws PlenigoException {
+        PlenigoManager.get().configure("https://api.s-devops.com", "AMXzF7qJ9y0uuz2IawRIk6ZMLVeYKq9yXh7lURXQ", "h7evZBaXvhaLVHYRue7X", false
```
Comment (maximilianschweitzer) [inline src/test/java/com/plenigo/sdk/services/AppManagementServiceTest.java]: Again? ;-(

File: src/test/java/com/plenigo/sdk/services/AppManagementServiceTest.java [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+import java.net.HttpURLConnection;
+import java.util.List;
+
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * Tests for {@link com.plenigo.sdk.services.ProductService}.
+ */
+//@RunWith(PowerMockRunner.class)
```
Comment (maximilianschweitzer) [inline src/test/java/com/plenigo/sdk/services/AppManagementServiceTest.java]: Why is this commented out?

File: src/main/java/com/plenigo/sdk/models/CustomerAppRequest.java (hunk 1 above)
Comment (rtorres87) [inline src/main/java/com/plenigo/sdk/models/CustomerAppRequest.java]: fixed

File: src/test/java/com/plenigo/sdk/services/AppManagementServiceTest.java (hunk 3 above)
Comment (rtorres87) [inline src/test/java/com/plenigo/sdk/services/AppManagementServiceTest.java]: i commited over this, i was doing the integration tests on the java-sdk project for faster testing.

File: src/test/java/com/plenigo/sdk/services/AppManagementServiceTest.java (hunk 2 above)
Comment (rtorres87) [inline src/test/java/com/plenigo/sdk/services/AppManagementServiceTest.java]: D:

**Pull request closed without merging**